Kickstarter-iOS/Views/Cells/PledgeShippingLocationCell.swift

```diff
@@ -76,10 +78,21 @@ final class PledgeShippingLocationCell: UITableViewCell, ValueCell {
 
   // MARK: - Configuration
 
-  func configureWith(value: (location: String, currency: String, rate: Double)) {
-    self.countryButton.setTitle(value.location, for: .normal)
-    self.amountLabel.text = "+\(value.currency)\(value.rate)"
+  func configureWith(value: PledgeDataSource.PledgeInputRow) {
```

Comment:
Maybe we should only allow this to be configured with the values we expect? That could also be a `typealias` in the associated value like `PledgeShippingLocationData`?

Reply:
Yeah I think this is a fair point. We share data source value types in `ProjectPamphletContentDataSource` (see `ProjectPamphletSubpage`), but the key difference is that the value configures two instances of *the same* cell (`ProjectPamphletSubpageCell`). In this case there are two *different* cells, which might be an argument for different `typealiased` value types. What I liked about using the same `enum` type though is being able to easily identify the type of cell at any given row, but I'll see if there's another way to do that.

Reply:
After chatting with @dusi we're thinking it makes more sense for these cells to have their own view models, to which we will directly pipe this values via a `configure(with: value)` input. In the view model, we can retrieve the associated values. This work will be part of a separate PR, so line 85 here is temporary, as are lines 89-92.